model/task_lifecycle.go

```diff
@@ -1281,13 +1274,16 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller, status string
 }
 
 // evalBisectStepback performs bisect stepback on the task.
-func evalBisectStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
+func evalBisectStepback(ctx context.Context, t *task.Task, caller string, stepback, deactivatePrevious bool) error {
 	// If the task is aborted or it isn't a mainline commit or existing stepback, no-op.
```

Comment:
Now that we're passing the variable in, we are also going to need to return nil if stepback is not enabled, just [as linear stepback does](https://github.com/evergreen-ci/evergreen/pull/7124/files/4e7eaa3469be3dc5647347961051a496d4648684..50facde7df89b085ef2b29648fb1be3c503f51a3#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9R1244-R1246). We should also update this comment

Reply:
oops.... totally missed adding that, my bad!

Reply:
np!